vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/dif/DocumentDigestEntry.kt

```diff
@@ -0,0 +1,158 @@
+@file:UseSerializers(UrlSerializer::class)
+
+package at.asitplus.wallet.lib.data.dif
+
+import at.asitplus.KmmResult
+import at.asitplus.KmmResult.Companion.wrap
+import io.ktor.http.*
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.UseSerializers
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+@Serializable
+data class DocumentDigestEntry private constructor(
+    val label: String,
+    /**
+     * base64 encoded octet representation generated using hashAlgorithmOID
+     */
+    val hash: String? = null,
+    val hashAlgorithmOID: String? = null,
+    @SerialName("documentLocation_uri") //errorous in test vector - changed for test case
```

Comment:
So that will be changed later?

Reply:
Left-over comment from before i started fixing the test vector because of bigger errors. New test vector in next draft